inference.Rmd

```diff
@@ -663,7 +691,7 @@ sample *is* the population, and take more samples (with replacement) of the
 same size from it instead! This very clever technique is 
 called **the bootstrap**.  Note that by taking many samples from our single, observed
 sample, we do not obtain the true sampling distribution, but rather an
-approximation that we call **the bootstrap distribution**. 
```

Comment:
above remove bolding for the word "point estimate" at very start of this section since it was defined already

Reply:
done